libs/k8s-client/src/BaseApi/PodWithLogStream.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Keboola\K8sClient\BaseApi;
+
+use GuzzleHttp\Handler\CurlHandler;
+use GuzzleHttp\HandlerStack;
+use Kubernetes\API\Pod;
+use KubernetesRuntime\Client;
+use Psr\Http\Message\ResponseInterface;
+
+class PodWithLogStream extends Pod
+{
+    public function __construct($namespace = 'default')
+    {
+        $this->client = Client::getInstance();
+        $this->namespace = $namespace;
+    }
+
+    public function readLog(string $namespace, string $name, array $queries = [])
+    {
+        $handler = new CurlHandler();
```

Comment:
Nejaky spesl duvod, proc tu pouzivas prave `CurlHandler` a nenechas Guzzle si vybrat?

Reply:
Okopíroval jsem to z dokumentace, jestli to jde nastavit nějak na automatický výběr tak budu rád za radu.

Reply:
No jednoduse udelas `$handlerStack = HandlerStack::create();` :slightly_smiling_face:

Reply:
Mozna, kdyz uz tam budes hrabat, hod tam jeste nad comment ve smyslu `use custom handler stack so that response is not eager loaded by logger`

Reply:
`$handlerStack = HandlerStack::create();` přísahal bych že jsem to zkoušel a samotné to nestačilo, každopádně teď to funguje, takže to tam dám, díky